### PR TITLE
Fix aggregate response for zero connected commons

### PR DESCRIPTION
**Pull request by nss10**
Jira Ticket: [BRH-555](https://ctds-planx.atlassian.net/browse/BRH-555)

## Improvements
* WTS now fetches aggregate response from unconnected commons by making a request without an access token.(previously they were skipped)  

[BRH-555]: https://ctds-planx.atlassian.net/browse/BRH-555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: wts/tokens.py [hunk 1]
```diff
@@ -67,9 +67,13 @@ async def async_get_access_token(refresh_token):
     Return:
```
Comment (Avantol13) [inline wts/tokens.py:67]: docstring needs an update

**Review by Avantol13 — CHANGES_REQUESTED** [review]
Please add unit test(s) necessary to test this change in behavior

File: wts/tokens.py (hunk 1 above)
Comment (nss10) [reply wts/tokens.py:68]: Updated.

**Comment by nss10** [comment]
> Please add unit test(s) necessary to test this change in behavior

Thank you. Unit tests added. @Avantol13

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**